internal/builder/v1beta1/pod.go

```diff
@@ -12,6 +12,7 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+
```

Comment:
Was this added by accident?

Reply:
ah that was to make sure it failed before.. so.. yes and no 😛 I can revert 😬